build.sh

```diff
@@ -1,13 +1,10 @@
-git submodule init
-git submodule update
 mkdir build install
-git apply --stat python2.7.patch
+git submodule add https://github.com/pybind/pybind11 python/pybind11
```

Comment:
Regarding `pybind11`, IIUD running this `build.sh` you will get the `pybind11` code you need to compile files inside the `python` directory. However, this `build.sh` may not be run using other package manager tools like (`lcgcmake` or `spack`), where `pybind11` should be defined as an external dependency, so I think `CMakeLists.txt` should also look for this package in order to be found without explicitly defining `LD_LIBRARY_PATH` or `PYTHONPATH`.

What do you think? I don't really know how was this `pybind11` dependency handled before, so I'm thinking out loud.